src/test/java/seedu/linkedout/logic/parser/SearchCommandParserTest.java

```diff
@@ -41,6 +46,10 @@ public void parse_keywordMissing_throwsParseException() {
         // missing job keyword
         assertParseFailure(parser, " j/", String.format(
                 MESSAGE_INVALID_COMMAND_FORMAT, SearchCommand.MESSAGE_CONSTRAINTS));
+
+        // missing round keyword
+        assertParseFailure(parser, " r/", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, SearchCommand.MESSAGE_CONSTRAINTS));
```

Comment:
Should we include multiple keyword missing as well since command is multiple prefix matching now, just to cover the bases and make sure the prefix are not parse as Strings

Reply:
Noted. As discussed in the meeting, @goel-a will add them in when he does his task!